docs/source/install.rst

```diff
@@ -18,7 +18,7 @@ You should then be able to provision the Vagrant box.
 
     $ vagrant up
 
-The box will be provisioned with Python 3.5, a virtual environment with all
+The box will be provisioned with Python 3.6.5, a virtual environment with all
```

Comment:
Nit: How about saying something like "the latest Python 3.6 release" or "version" so we don't need to update the version information every six month :)

Reply:
Ok, will do.
I need a day or two though as was really busy today - work / training :).

Reply:
Sure, there is no rush :)